packages/app/src/cli/models/app/loader.ts

```diff
@@ -318,10 +320,13 @@ class AppLoader {
       )
     }
 
+    const customConfigurationObject =
```

Comment:
This was added so that app config specifications can read from the same app schema but only use parts of it.